infra/terraform/bootstrap/azure_devops.tf

```diff
@@ -13,7 +13,7 @@ resource "azuredevops_project" "project" {
   work_item_template = "Agile"
 
   lifecycle {
-    prevent_destroy = true # prevent destroying the project
+    #prevent_destroy = true # prevent destroying the project
```

Comment:
Just checking if we want to have this commented out in the bootstrap

Reply:
oh yes, commented on accident! Thank you!